packages/sw-lib/src/index.js

```diff
@@ -0,0 +1,35 @@
+/*
+ Copyright 2016 Google Inc. All Rights Reserved.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+/* eslint-env browser */
+/* eslint-disable no-console */
+
+import ErrorFactory from './lib/error-factory';
+import SWLib from './lib/sw-lib';
+import {Route} from '../../sw-routing/src/index.js';
+
+const initialiseLibrary = () => {
+  if (!('ServiceWorkerGlobalScope' in self &&
```

Comment:
Consider moving this sort of thing to https://github.com/GoogleChrome/sw-helpers/blob/master/lib/assert.js in case it's useful elsewhere.

It's kind of like a special-case form of the [`isInstance()`](https://github.com/GoogleChrome/sw-helpers/blob/master/lib/assert.js#L32) check.